platform/node/scripts/after_success.sh

```diff
@@ -4,7 +4,7 @@ set -e
 set -o pipefail
 
 if [[ -n ${PUBLISH:-} ]]; then
-    if [[ "${BUILDTYPE}" == "Release" ]]; then
+    if [[ "${BUILDTYPE}" == "RelWithDebInfo" ]]; then
```

Comment:
@springmeyer yeah I thought about this, but I'm sure if the code path `if [[ "${BUILDTYPE}" == "Release" ]]` would ever be hit. With this PR, we'd be removing `Release` completely.